circle.yml

```diff
@@ -9,6 +9,11 @@ checkout:
   post:
     # Update the `master` branch. Otherwise CircleCI will give us a cached one.
     - git fetch -u origin +master:master
+    # Checkout the merged PR for testing as CircleCI will just use the PR head otherwise.
+    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/head:pr/$CIRCLE_PR_NUMBER/head ; fi
+    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/merge:pr/$CIRCLE_PR_NUMBER/merge ; fi
+    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git checkout -qf pr/$CIRCLE_PR_NUMBER/merge ; fi
+    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git branch --merged | grep master > /dev/null ; fi
```

Comment:
This conflict detection mechanism feels kludgy. Seems like it could go wrong if `master` passed us. Will give this some more thought.